### DISCUSSION TITLE
Use inside of a Repeater component

### DISCUSSION THREAD

**Discussion by howdu** [camya/filament-title-with-slug#2] [Q&A] [answered]
Wondering if it's possible or a bug. I'm finding the autocomplete slug is not working after create. 

```php
Forms\Components\Repeater::make('descriptions')
    ->relationship('descriptions')
    ->collapsible()
    ->schema([
        TitleWithSlugInput::make(
            urlVisitLinkLabel: false,
            titleLabel: 'Title',
            fieldTitle: 'title',
            fieldSlug: 'slug',
        ),
```

**Comment by camya** [comment, marked as answer]
Hello, thank you very much for your report.

I have looked into it a bit and found a solution. The next version will contain the fix. (You can already test it in the main branch).

You can read the updated documentation: [Use inside a relationship repeater](https://github.com/camya/filament-title-with-slug#use-within-a-relationship-repeater)

Have fun.

**Reply by howdu** [reply]
Brilliant thank you - I can confirm the latest changes are working great inside a Repeater.

**Reply by camya** [reply]
Thank you for testing it. Have a nice day.
